tests/unit/bokeh/core/property/test_container.py

```diff
@@ -35,6 +35,7 @@
     'Dict',
     'List',
     'RelativeDelta',
+    'RestrictedDict',
```

Comment:
Is this not required?
Adding `RestrictedDict` throws an error, not adding it doesn't.